sdk/eventhub/event-hubs/samples/javascript/README.md

```diff
@@ -67,9 +68,9 @@ Take a look at our [API Documentation][apiref] for more information about the AP
 [usewithiothub]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-hubs/samples/javascript/useWithIotHub.js
 [websockets]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-hubs/samples/javascript/websockets.js
 [usingaadauth]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-hubs/samples/javascript/usingAadAuth.js
-[receiveeventsusingcheckpointstore]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-hubs/samples/javascript/receiveEventsUsingCheckpointStore.js
 [receiveevents]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-hubs/samples/javascript/receiveEvents.js
 [apiref]: https://docs.microsoft.com/javascript/api/@azure/event-hubs
+[checkpointing]: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/eventhubs-checkpointstore-blob/samples/receiveEventsUsingCheckpointStore.js
```

Comment:
This is going to a 404

Reply:
The file that link points to is in this commit, so it will not 404 once it is merged.